apps/guide/src/content/02-creating-your-bot/03-command-handling.mdx

```diff
@@ -0,0 +1,220 @@
+---
+title: Command handling
+category: Creating your bot
+---
+
+# Command handling
+
+Unless your bot project is small, it's not a very good idea to have a single file with a giant _`if`_/_`else if`_ chain for commands. If you want to implement features into your bot and make your development process a lot less painful, you'll want to implement a command handler. Let's get started on that!
+
+<Alert title="Tip" type="success">
+For fully functional slash commands, there are three important pieces of code that need to be written. They are:
+
+1. The [individual command files](slash-commands), containing their definitions and functionality.
+2. The command handler, which dynamically reads the files and executes the commands.
+3. The [command deployment script](command-deployment), to register your slash commands with Discord so they appear in the interface.
+
+These steps can be done in any order, but **all are required** before the commands are fully functional.
+
+This page details how to complete **Step 2**. Make sure to also complete the other pages linked above!
+
+</Alert>
+
+## Loading command files
+
+Now that your command files have been created, your bot needs to load these files on startup.
+
+In your _`index.js`_ file, make these additions to the base template:
+
+<CH.Code>
+
+```js index.js focus=1:2,8
+import { readdirSync } from 'node:fs';
+import { join } from 'node:path';
+// focus[18:27]
+import { Client, Collection, Events, GatewayIntentBits } from 'discord.js';
+import config from './config.json';
+
+const client = new Client({ intents: [GatewayIntentBits.Guilds] });
```

Comment:
https://github.com/discordjs/discord.js/blob/0d0e4d1cb61e27d476352294a5624fec2bbf657b/apps/guide/src/content/02-creating-your-bot/01-initial-files.mdx?plain=1#L201

Should probably put that inside an array or use `|`?

Reply:
Yeah but that's a different file? Kinda out of scope of this PR